ls1intum/Artemis
# Enhance instructors exam checklist table

### Checklist

### Motivation and Context



### Description

We enhanced the instructors exam checklist table by reorganizing the way that the exercise group informations are presented. We combined checks into one table. If a group has problems, a orange warning will appear next to the group.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration -> exam -> checklist
3. Check if the informations are displayed correctly. 

### Test Coverage



ExamChecklistExerciseGroupTableComponent: 95%

### Screenshots


![Bildschirmfoto 2021-03-10 um 17 49 30](https://user-images.githubusercontent.com/42205689/110665807-03608980-81c9-11eb-8ccc-8746e2691438.png)
